feat: add parsing of argument type by default value

ifedchankau opened the pull request:
- Add parsing of argument type by default value
- Change types of default values for numbers & booleans

Closes #151

ghost commented:
There were the following issues with this Pull Request

* Commit: 9e7ff6d3299069ae2b01bb736863b800c5e13a45
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues

ifedchankau committed:
feat: add parsing of argument type by default value

- Add parsing of argument type by default value
- Change types of default values for numbers & booleans

Closes #151

romanrostislavovich reviewed (approved):
+1

romanrostislavovich commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@linterhub/usage-parser)
- [GitHub release](https://github.com/linterhub/usage-parser/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
